projects/1.16/assets/dustrial-decor/dustrial_decor/lang/zh_cn.json

```
@@ -1 +1,114 @@
+    "block.dustrial_decor.rusty_sheet_metal_treading_slab": "生锈的钣金踏面台阶",
+    "block.dustrial_decor.rusty_sheet_metal_treading_vertical_slab": "生锈的钣金踏面竖直台阶",
+    "block.dustrial_decor.rusty_sheet_metal_treading_stairs": "生锈的钣金踏面楼梯",
+    "block.dustrial_decor.rusty_sheet_metal_trapdoor": "生锈的钣金活板门",
+    "block.dustrial_decor.rusty_sheet_metal_door": "生锈的钣金门",
+    "block.dustrial_decor.cinder_block": "煤渣块",
+    "block.dustrial_decor.polished_cinder_block": "抛光煤渣块",
+    "block.dustrial_decor.cinder_bricks": "煤渣砖",
+    "block.dustrial_decor.cinder_brick_slab": "煤渣砖台阶",
+    "block.dustrial_decor.cinder_brick_vertical_slab": "煤渣砖竖直台阶",
+    "block.dustrial_decor.cinder_brick_stairs": "煤渣砖楼梯",
+    "item.dustrial_decor.cardboard": "纸板",
+    "block.dustrial_decor.cardboard_box": "纸箱",
+    "block.dustrial_decor.folded_cardboard": "折叠纸板",
+    "block.dustrial_decor.smooth_cardboard": "平滑纸板",
+    "block.dustrial_decor.smooth_cardboard_slab": "平滑纸板台阶",
+    "block.dustrial_decor.smooth_cardboard_vertical_slab": "平滑纸板竖直台阶",
+    "block.dustrial_decor.smooth_cardboard_stairs": "平滑纸板楼梯",
+    "block.dustrial_decor.cardboard_scrap": "纸板废料",
+    "block.dustrial_decor.cardboard_trapdoor": "纸板活板门",
+    "block.dustrial_decor.cardboard_door": "纸板门",
+    "block.dustrial_decor.rusty_iron_block": "生锈的铁块",
+    "item.dustrial_decor.rusty_iron_ingot": "生锈的铁锭",
+    "item.dustrial_decor.rusty_iron_nugget": "生锈的铁粒",
+    "block.dustrial_decor.wrapped_chains": "包裹的锁链",
+    "block.dustrial_decor.wrapped_gold_chains": "包裹的金链",
+    "block.dustrial_decor.rusty_iron_door": "生锈的铁门",
+    "block.dustrial_decor.rusty_iron_trapdoor": "生锈的铁活板门",
+    "block.dustrial_decor.iron_bar_door": "铁条门",
+    "block.dustrial_decor.iron_bar_trapdoor": "铁条活板门",
+    "block.dustrial_decor.gold_chain": "金链",
+    "block.dustrial_decor.large_chain": "粗铁链",
+    "block.dustrial_decor.large_gold_chain": "粗金链",
+    "block.dustrial_decor.anchor": "锚索",
+    "block.dustrial_decor.hook": "钩子",
+    "block.dustrial_decor.redstone_lantern": "红石灯笼",
+    "block.dustrial_decor.large_lantern": "大型灯笼",
+    "block.dustrial_decor.large_soul_lantern": "大型灵魂灯笼",
+    "block.dustrial_decor.large_redstone_lantern": "大型红石灯笼",
+    "block.dustrial_decor.red_neon_light": "红色霓虹灯",
+    "block.dustrial_decor.orange_neon_light": "橙色霓虹灯",
+    "block.dustrial_decor.yellow_neon_light": "黄色霓虹灯",
+    "block.dustrial_decor.green_neon_light": "绿色霓虹灯",
+    "block.dustrial_decor.cyan_neon_light": "青色霓虹灯",
+    "block.dustrial_decor.blue_neon_light": "蓝色霓虹灯",
+    "block.dustrial_decor.pink_neon_light": "粉红色霓虹灯",
+    "block.dustrial_decor.purple_neon_light": "紫色霓虹灯",
+    "block.dustrial_decor.rainbow_neon_light": "彩虹霓虹灯",
+    "block.dustrial_decor.black_light": "黑灯",
+    "block.dustrial_decor.large_ender_lantern": "大型末影灯笼",
+    "block.dustrial_decor.large_cursed_lantern": "大型诅咒灯笼",
+    "block.dustrial_decor.large_glow_lantern": "大型辉光灯笼",
+    "block.dustrial_decor.large_ice_chain": "大型冰链",

```

```suggestion
    "block.dustrial_decor.large_ice_chain": "粗冰链",
```
这里应该统一名称吧，之前是`粗铁链` `粗金链`